### PR TITLE
add June 2021 blog

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

**Comment by dholbach** [comment]
cf https://deploy-preview-305--fluxcd.netlify.app/blog/2021/05/june-2021-update/

**Comment by dholbach** [comment]
waiting for tomorrow's Flagger release to be included - please don't merge - still WIP

**Comment by dholbach** [comment]
added Flagger 1.11

File: content/en/blog/2021-05-31-june-update.md
Comment (scottrigby) [inline content/en/blog/2021-05-31-june-update.md:5]: I'm thinking the short phrase "we are reviewing Flux governance" could be interpreted to mean something different – like perhaps we are rethinking governance, or it wasn't fully established yet, etc. While a 6 month review feels more planned and standard, as is the case. WDYT?

```suggestion
description: Flux 0.14 and Flagger 1.11 with lots of new features, KubeCon talks galore, Flux at CNCF Store, lots of Flux adoption. Flux governance 6 month review.
```

File: content/en/blog/2021-05-31-june-update.md
Comment (scottrigby) [inline content/en/blog/2021-05-31-june-update.md:26]: Perhaps link to the Helm chart
```suggestion
  [kube-prometheus-stack](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack).
```

File: content/en/blog/2021-05-31-june-update.md
Comment (scottrigby) [inline content/en/blog/2021-05-31-june-update.md:90]: Do we want to add all Kubecon talks by Flux maintainers? If we just want to list a few of them, that is fine, we can probably remove this line entirely

Comment (dholbach) [reply content/en/blog/2021-05-31-june-update.md:90]: Do you have more links?

Comment (dholbach) [reply content/en/blog/2021-05-31-june-update.md:90]: I went with what's on https://fluxcd.io/community/#talks

Comment (scottrigby) [reply content/en/blog/2021-05-31-june-update.md:90]: I know we listed them all in the May blog 🙂

**Review by scottrigby — APPROVED** [review]
❤️

**Comment by dholbach** [comment]
Thanks for the reviews and edits! :gift:

**Pull request merged**
